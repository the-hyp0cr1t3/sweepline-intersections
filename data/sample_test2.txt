3
1 1 9 9
1 1 9 -9
4 7 6 -7