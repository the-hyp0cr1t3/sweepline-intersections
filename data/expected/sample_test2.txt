3
1.0000000000 1.0000000000 1 2
4.3750000000 4.3750000000 1 3
5.6956521739 -4.8695652174 2 3

